chapter_computational-performance/hardware_vn.md

```diff
@@ -336,13 +340,21 @@ Often the latter is capable of performing many operations simultaneously.
 For instance, the ARM Cortex A77 core of :numref:`fig_cortexa77` is able to perform up to 8 operations simultaneously.
 -->
 
-*dịch đoạn phía trên*
+Mỗi nhân xử lý bao gồm các thành phần rất tinh vi.
+Mặc dù chi tiết khác nhau giữa đời máy và nhà sản xuất, chức năng cơ bản của chúng tương đối đã được chuẩn hoá.
+Front-end tải các lệnh và dự đoán nhánh nào sẽ được thực hiện (ví dụ: cho luồng điều khiển).
+Các lệnh tiếp đó được giải mã từ mã nguồn hợp ngữ (assembly code) thành vi lệnh.
+Mã nguồn hợp ngữ thường chưa phải là mã nguồn cấp thấp nhất mà bộ xử lý thực thi.
+Thay và đó, các lệnh phức tạp có thể được giải mã thành một tập hợp các hoạt động cấp thấp hơn.
+Tiếp đó chúng được xử lý bằng một nhân thực thi thực .
+Thông thường bộ xử lý sau có khả năng thực hiện đồng thời nhiều thao tác.
+Ví dụ, nhân ARM Cortex A77 trong :numref:`fig_cortexa77` có thể thực hiện lên đến 8 thao tác cùng một lúc.
```

Comment:
```suggestion
Thông thường bộ xử lý có khả năng thực hiện đồng thời nhiều phép toán.
Ví dụ, lõi ARM Cortex A77 trong :numref:`fig_cortexa77` có thể thực hiện lên đến 8 phép toán cùng một lúc.
```

Reply:
`phép toán` hay `phép tính` nhỉ @cuongvng , liên quan tới máy tính trước giờ mình chỉ hay dịch `phép tính` thôi. 
```suggestion
Các bộ xử lý đời mới thường có khả năng thực hiện đồng thời nhiều câu lệnh.
Ví dụ, lõi ARM Cortex A77 trong :numref:`fig_cortexa77` có thể thực hiện lên đến 8 phép tính cùng một lúc.
```

Reply:
well em hiểu cả hai như nhau. Cơ mà cho CPU thì các `operation` ở mức đơn giản như + - * thì dịch là `phép tính` hay hơn.